models/DataObject/Localizedfield.php

```diff
@@ -635,6 +635,9 @@ public function __sleep()
             foreach ($lazyLoadedFields as $fieldName) {
                 foreach (Tool::getValidLanguages() as $language) {
                     unset($this->items[$language][$fieldName]);
+
+                    $lazyKey = $fieldName . LazyLoadedFieldsInterface::LAZY_KEY_SEPARATOR . $language;
```

Comment:
please refactor `$lazykey` generation to a separate method since used at many places. thanks!